ISSUE_TEMPLATE.md

```diff
@@ -1,15 +1,15 @@
 #### Description of feature or bug
 
-TODO
+<TODO>
```

Comment:
putting the angle brackets around these makes them disappear when the markdown is rendered since they're interpreted as a DOM element. Is that what you were going for?

Reply:
If they show up in the "new issue" screen but not in the final published screen, that seems idea.

Reply:
that way they don't take up space unless filled in